### ISSUE TITLE
AdSense setup error

### ISSUE THREAD

**Issue by MertcanGokgoz** [google/site-kit-wp#2106]
## Bug Description

Wordpress REST API works well but google-site-kit links do not work and returns 400 error. 

I am getting errors in the links below.(400 HTTP)

https://example.com/wp-json/google-site-kit/v1/modules/adsense/data/clients?accountID=pub-12345&_locale=user
https://example.com/wp-json/google-site-kit/v1/modules/adsense/data/alerts?accountID=pub-12345&_locale=user

but when I looked at the link, came the following error.

```
{"code":"rest_forbidden","message":"\u00dczg\u00fcn\u00fcm, bunu yapman\u0131za izin verilmiyor.","data":{"status":401}}
```

I could not understand the problem. While the api is running smoothly, I am getting an error. I activated other modules.

## Steps to reproduce

1. Go to 'wordpress admin area'
2. Click on 'google site kit > settings'
3. Scroll down to 'settings > connected services > adsense > continue module setup'

## Screenshots

![](https://i.imgur.com/X06xvx9.png)

## Additional Context

 - PHP Version: 7.4.10
 - OS: Windows 10 Pro Version 2004
 - Google Chrome Version 85.0.4183.121 (Official Build) (64-bit)
 - Plugin Version 1.16.0
 - Device: Computer

plugins i use

```
AddToAny Share Buttons | Sürüm: 1.7.42, geliştirici: AddToAny \| Otomatik güncellemeler etkin
AMP | Sürüm: 2.0.4, geliştirici: AMP Project Contributors \| Otomatik güncellemeler etkin
Broken Link Checker | Sürüm: 1.11.14, geliştirici: WPMU DEV \| Otomatik güncellemeler etkin
Compress JPEG & PNG images | Sürüm: 3.2.1, geliştirici: TinyPNG \| Otomatik güncellemeler etkin
Disable Search | Sürüm: 1.8.1, geliştirici: Scott Reilly \| Otomatik güncellemeler etkin
DMCA Website Protection Badge | Sürüm: 2.0.4, geliştirici: DMCA.com \| Otomatik güncellemeler etkin
Instant Indexing | Sürüm: 1.0.0, geliştirici: Rank Math \| Otomatik güncellemeler etkin
Neori Shortcodes | Sürüm: 1.0, geliştirici: litMotion \| Otomatik güncellemeler etkin
pipDisqus | Sürüm: 1.4, geliştirici: pipdig \| Otomatik güncellemeler etkin
Rank Math SEO | Sürüm: 1.0.48, geliştirici: Rank Math \| Otomatik güncellemeler etkin
RSS Featured Image | Sürüm: 1.0.5, geliştirici: Jordy Meow \| Otomatik güncellemeler etkin
Site Kit by Google | Sürüm: 1.16.0, geliştirici: Google \| Otomatik güncellemeler etkin
Super Progressive Web Apps | Sürüm: 2.1.2, geliştirici: SuperPWA \| Otomatik güncellemeler etkin
Two Factor | Sürüm: 0.7.0, geliştirici: Plugin Contributors \| Otomatik güncellemeler etkin
Ultimate Addons for Gutenberg | Sürüm: 1.17.0, geliştirici: Brainstorm Force \| Otomatik güncellemeler etkin
W3 Total Cache | Sürüm: 0.15.0, geliştirici: BoldGrid \| Otomatik güncellemeler etkin
WP Offload SES Lite | Sürüm: 1.4.3, geliştirici: Delicious Brains \| Otomatik güncellemeler etkin
```

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## QA Brief

* 

## Changelog entry

*

**Comment by jamesozzie** [comment]
@MertcanGokgoz Thanks for reaching out. Initial support requests can be submitted using the plugins WordPress support forum. 

If you open a [new support topic](https://wordpress.org/support/plugin/google-site-kit/#new-post) there while providing your Site Health information using [this form](https://docs.google.com/forms/d/1lhjns2K7c0Ny7ME4gxTcsnYtNUtcMr8BvFpPQi23Y1E/) we'd be happy to assist. 

Note also I checked your sites REST endpoints and I encountered a `{"code":"rest_login_required","message":"REST API restricted to authenticated users` notice. I wonder are you using a plugin that has some REST modification features. Let's troubleshoot from the support topic and revisit here if needed.
